controllers/enrollmentkeys.go

```diff
@@ -155,7 +165,10 @@ func handleHostRegister(w http.ResponseWriter, r *http.Request) {
 		logic.ReturnErrorResponse(w, r, logic.FormatError(err, "internal"))
 		return
 	}
+	// set defaults
+	newHost.PersistentKeepalive = models.DefaultPersistentKeepAlive
```

Comment:
only set this for a new host, check `hostexists` check below

Reply:
https://github.com/gravitl/netmaker/pull/2604/commits/fcc150aad648ae394cf471046368b02416be1a97